src/components/item-details/place-bet.tsx

```diff
@@ -8,13 +8,13 @@ import { useState } from 'react';
 
 type Props = {
   highestBid?: HighestBid;
-  auctionDate?: string;
+  actionDate?: string;
```

Comment:
Minor. Not sure if we need this page, but the previous name was correct, it means 'subasta', so auction is correct.

Reply:
Oh, okay. I got it